sixpack.js

```diff
@@ -65,6 +66,16 @@
 
     sixpack.Session.prototype = {
         participate: function(experiment_name, alternatives, traffic_fraction, force, callback) {
+            if (this.debug) {
+                console.log('SixpackSession - Participate Params', {
+                    experiment_name,
+                    alternatives,
+                    traffic_fraction,
+                    force,
+                    callback,
+                    timeout: this.timeout,
+                });
+            }
```

Comment:
Instead of, every time, check `this.debug` and open a conditional block, you could create a private method (oh my god JS) called `log` and let this method check the `if (this.debug)` and call the `console.log` with a given message